tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -58,16 +62,26 @@ module('Acceptance | guid file | registration files', hooks => {
         assert.dom('[data-test-tags-button]').exists('Tags button exists');
         await percySnapshot('Acceptance | guid file | registration files | mobile view | file renderer');
 
+        assert.dom('[data-test-file-renderer]').isVisible();
+        assert.dom('[data-test-metadata-tab]').isNotVisible();
+        assert.dom('[data-test-revisions-tab]').isNotVisible();
+        assert.dom('[data-test-tags-tab]').isNotVisible();
+
+        // click tags and verify appropriate panels shown
         await click('[data-test-versions-button]');
         assert.dom('[data-test-revisions-tab]').exists('Revisions shown');
-        assert.dom('[data-test-file-renderer]').doesNotExist('File renderer is hidden');
+        assert.dom('[data-test-file-renderer]').isNotVisible('File renderer is hidden');
         await percySnapshot('Acceptance | guid file | registration files | mobile view | revisions');
 
-        // click tags and verify appropriate panels shown
-
         await click('[data-test-file-renderer-button]');
         assert.dom('[data-test-file-renderer]').exists('File renderer is shown again');
-        assert.dom('[data-test-revisions-tab]').doesNotExist('Revisions now hidden');
+        assert.dom('[data-test-revisions-tab]').isNotVisible('Revisions now hidden');
+
+        await click('[data-test-metadata-button]');
+        assert.dom('[data-test-metadata-tab]').exists('Metadata tab now visible');
```

Comment:
Same here on `data-test-metadata-tab`.

Reply:
Yes, I have updated it with the divider and data list structure. Let me know if this is ok.